index.js

```diff
@@ -12,68 +12,191 @@
+                // print the iteration until the loop ends
+                console.log(i)
+            }
+        }
+    }
+    return dollars
+}
+console.log(showMeTheMoney(200))
+/*
+ @Challenge 04 - Write a function named `showTheFifties` that loops from 1 to 200.
  Use a condition in the loop to console.log the number when it is over 49 and under 60
  @Example - Expect to see 50, 51, 52, 53, 54, 55, 56, 57, 58, 59 in the terminal
  @Test - Make a function call of  `showTheFifties()`
 */
-
-/* 
+function showTheFifties(vintageItem) {
+    // if variable is greater than 0, continue to for loop
+    if (vintageItem > 0) {
+        // loop 0 to 200, increasing by an increment of 1
+        for (let i = 0; i < 200; i++) {
+            // if the iteration is between the numbers 49 and 60
+            if (i > 49 && i < 60)
+                // print those numbers, then continue increasing without
+                // printing the numbers 60 to 200
+                console.log(i)
+        }
+    }
+    return vintageItem
+}
+console.log(showTheFifties(200))
+/*
  @Challenge 05 - Write a function that returns an array that is named `getTheTwenties` that loops from 1 to 60 and returns an array of all the 20's
  Use a condition in the loop to push to an array when the number is greater than or equal to 20 and under 30
  @Example - Expect an array returned with value of [20, 21, 22, 23, 24, 25, 26, 27, 28, 29]
  @Test - console.log the result of a function call of  `getTheTwenties()` and expect to see an array value of [20, 21, 22, 23, 24, 25, 26, 27, 28, 29]
 */
-
-/* 
+function getTheTwenties(twenties) {
+    // create a new array
+    let arr = [];
+    // set the start and end points for ease of reading
+    let start = 20;
+    let end = 30;
+    // loop through the array 60 times
+    for (let i = 0; i < 60; i++) {
+        // when the iteration hits 20, push it to the new array
+        // do this for all iterations until instructed to end
+        // when the iteration hits 30, do not push to the array
+        if (i > start && i < end) {
+            arr.push(i);
+        }
+    }
+    return arr;
+}
+console.log(getTheTwenties(60))
+/*
  @Challenge 06 - Write a function that accepts an array names. Name the function `sayHiToJim` that loops through the array of names.
  Use a condition in the loop to check if the name is "Jim". If the name is "Jim" console.log a message to say "Hi Jim"
  Use conditional logic to console.log "Ignoring " and then the name of the person being ignored.
  @Example - Expect to see "Ignorning Jane" and "Ignoring Anita Bath" when the function is given an array of ["Jane", "Anita Bath"]
  @Test01 - Make a function call `sayHiToJim(["Jane", "Anita Bath"])`
  @Test02 - Make a function call `sayHiToJim(["Jane", "Anita Bath", "Jim", "Sam Sung"])`
 */
+// THIS FUNCTION KEEPS GETTING UNDEFINED FROM JANE AND ANITA'S ARRAY
+function sayHiToJim(nameList) {
+    // looping through each array's list of names
```

Comment:
`i` is not being declared with a `let` here. This may result in a bug in the code. Be sure to declare using `let`